frontend/src/js/Header/index.jsx

```diff
@@ -9,8 +9,12 @@ class Header extends React.Component {
       <div>
         <Link to='/'><h1>Amazon 2.0 Home Page</h1></Link>
         <header style={styles}>
-          <SearchBar submitHandler={this.props.search} categorySelect={this.props.categorySelect} handleChange={this.props.handleChange} />
-          <Nav home={'/home'} browse={'/search'} checkout={'/checkout'} basket={'/basket'}/>
+          <SearchBar
+            submitHandler={this.props.search}
+            categorySelect={this.props.categorySelect}
+            handleChange={this.props.handleChange}
+            setResultsState={this.props.setResultsState} />
+          <Nav home={'/home'} browse={'/search'} checkout={'/payment'} basket={'/basket'}/>
```

Comment:
checkout here needs to stay as '/checkout' I think. The payment was not working when checkout={'/payment'}